modules/postgresql/versions.tf

```diff
@@ -27,19 +27,19 @@ terraform {
     }
     google = {
       source  = "hashicorp/google"
-      version = ">= 5.6, < 6"
+      version = ">= 5.12, < 6"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = ">= 5.6, < 6"
+      version = ">= 5.12, < 6"
     }
   }
 
   provider_meta "google" {
-    module_name = "blueprints/terraform/terraform-google-sql-db:postgresql/v19.0.0"
+    module_name = "blueprints/terraform/terraform-google-sql-db:postgresql_with_failover/v19.0.0"
```

Comment:
Can we keep this as is since this will create an unnecessary dip in the metrics? As long as we have the features documented in the readme, it should be fine to keep the name as is.